components/tensorflow-notebook-image/releaser/components/workflows.libsonnet

```diff
@@ -124,10 +122,26 @@
         },
         sidecars: sidecars,
       };  // buildTemplate
-
-      local buildImageTemplate(step_name, image, base_image, tf_package) =
-        buildTemplate(
-          step_name,
+      local buildImageTemplate(tf_version, device) = {
+        local image = params.registry + "/tensorflow-notebook-" + device + ":" + tf_version + "-" + params.versionTag,
```

Comment:
nil: I think the tf_version should be in the image name not the tag
e.g
"tensorflow-notebook-"+ tf_version + "-" device + ":" + params.versionTag

Reply:
Done